omnibus/config/software/system-probe.rb

```diff
@@ -21,6 +21,53 @@
 
   if ENV.has_key?('SYSTEM_PROBE_BIN') and not ENV['SYSTEM_PROBE_BIN'].empty?
     copy "#{ENV['SYSTEM_PROBE_BIN']}/system-probe", "#{install_dir}/embedded/bin/system-probe"
+
+    mkdir "#{install_dir}/.debug"
```

Comment:
this part will be rewritten, but that's the idea to do that here
as It would be done when agent, security-agent, process-agent and system-probe binary are available
I would need to have the binary signature after strip pass, but didn't find a way to hook (or run later, but before the packager) the Stripper on omnibus